torch/__init__.py

```diff
@@ -322,6 +322,7 @@ def storage_type(cls):
 ################################################################################
 
 import torch.cuda
+import torch.cuda.sparse
```

Comment:
I think a better place for that import would be in torch.cuda (from . import sparse).